### ISSUE TITLE
PDF have blue background

### ISSUE THREAD

**Issue by pscom** [barryvdh/laravel-dompdf#334]
Hello guys.

I didn't find issue like that soo... I'm writing.
When my app generete PDF the document have in some places blue background color like on image:

![bez tytulu](https://cloud.githubusercontent.com/assets/5529578/24532714/c57eaea6-15c2-11e7-8728-6283c465c808.png)

In my app I'm using Laravel and CSS Framework [Semantic UI](https://semantic-ui.com/).

Somebody know, what is wrong? :p

**Comment by rgb-one** [comment]
something similar for me but the blue background without any text. Without stylesheet added the data shows fine. But when I add a <link> to the main stylesheet in the markup, nothing but the blue background shows. I'm using Laravel as well with Bootstrap as the CSS Framework.
[document (1).pdf](https://github.com/barryvdh/laravel-dompdf/files/900895/document.1.pdf)

**Comment by lorisleiva** [comment]
Same problem with Semantic UI

**Comment by rgb-one** [comment]
This issue was due to bootstrap styles being used in the pdf template. You
can fix your issue by excluding the Samantic UI styles  in the template
used by dom pdf and add your own cusom inline styles. You can also override
the symantic UI styles like setting background-color: transparent;, etc.

TLDR;
Symantic UI sets the backgound color to blue; either remove or override the
Symantic UI styles for the template.
